evap/locale/de/LC_MESSAGES/django.po

```diff
@@ -34,7 +34,7 @@ msgstr "Name (Deutsch)"
 msgid "Name (English)"
 msgstr "Name (Englisch)"
 
-#: evap/contributor/forms.py:54 evap/staff/forms.py:318
+#: evap/contributor/forms.py:54 evap/staff/forms.py:319
```

Comment:
Regarding the translation: I ran `./manage.py makemessages -l de` which updated very many code line refernces that got changed sometime in the past. Do you prefer manually adding the new translations or is the big "update" done somewhere else once in a while?

Reply:
we do the translations on every update of the release branch. so you only have to make sure that all strings are marked for translation in views (`_("translate this")`) and templates (`{% trans 'translate this' %}`) and i'll do the rest. please do not commit any changes on the `django.po` file.